AdminUi/src/AdminUi.Infrastructure/Persistence/Database/EntityTypeConfigurations/IdentityOverviewEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using AdminUi.Infrastructure.DTOs;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace AdminUi.Infrastructure.Persistence.Database.EntityTypeConfigurations;
+
+public class IdentityOverviewEntityTypeConfiguration : IEntityTypeConfiguration<IdentityOverviewDTO>
+{
+    public void Configure(EntityTypeBuilder<IdentityOverviewDTO> builder)
+    {
+        builder.ToView("IdentityOverviews");
+        builder.HasKey(i => i.Address);
```

Comment:
We should either define a key in the view itself as well, if that is possible; or we should use `builder.HasNoKey()` here. But it's probably not a good idea to lie to EF Core.

Reply:
Isn't it better for performance reasons to have a key?